tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -378,3 +391,18 @@ def test_date_range_bounds_kwarg_default_monday(placeholder_transform):
 def test_format_date_range(placeholder_transform, date_range, expected):
     actual = placeholder_transform.format_date_range(date_range)
     assert actual == expected
+
+
+@pytest.mark.parametrize(
+    "answer_id,value,expected",
+    [
+        ("mandatory-checkbox-answer", "Head", "Head label"),
+        ("mandatory-checkbox-answer", "Body", "Body label"),
+    ],
+)
+def test_option_label_from_value_with_placeholder_label(
```

Comment:
We should unit test most of it in operation.py. We also need to unit test with labels that contain placeholders. The unit test here could just be for coverage assuming it is reusing the function from operations.py.